test/match-test.js

```diff
@@ -440,6 +440,52 @@ describe("sinonMatch", function () {
         });
     });
 
+    describe(".in", function () {
+        it("returns matcher", function () {
+            var inMatcher = sinonMatch.in([]);
+            assert(sinonMatch.isMatcher(inMatcher));
+        });
+
+        context("when the test is called with non array argument", function () {
+            it("throws TypeError (array expected)", function () {
+                var arg = "not-array";
+
+                assert.exception(function () {
+                    sinonMatch.in(arg);
+                }, {name: "TypeError"});
```

Comment:
It would be good to also test for the message, so we can be sure that it's actually expected error. Otherwise, another error might go undetected by this test for a long time.